### PR TITLE
Added express server, client and static page

### PR DESCRIPTION
**Pull request by AmiraVomir**
@carlitoswillis please check thank you

### CODE REVIEW

File: client/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './src/App.jsx'
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (carlitoswillis) [inline client/index.js:6]: thought we agreed not to use App?

File: client/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <div>
+        <div>
+          hello from react
+        </div>
+      </div>
+    )
+  }
+}
+
+export default App;
```
Comment (carlitoswillis) [inline client/src/App.jsx:20]: add a return at the end

File: server/index.js [hunk 3]
```diff
@@ -1 +1,12 @@
...
+const path = require('path');
+
+const port = 3003;
+
+
+app.use(express.static(path.join(__dirname, '../public')))
+
+app.listen(port, () => {
+  console.log(`listening on ${port}`)
+});
```
Comment (carlitoswillis) [inline server/index.js:12]: return

File: client/index.js (hunk 1 above)
Comment (AmiraVomir) [reply client/index.js:6]: from the lecture video, it looks as though the what you call it doesn't matter. Fred had 4 different src calling 4 different api end points called App. Can change if you want

Comment (carlitoswillis) [reply client/index.js:6]: not a big deal at all just curious! its all approved :+)

**Comment by AmiraVomir** [comment]
> don't forget those new lines at the end of the files. you also might want to remove package-lock.json and package.json from your repo and add them to .gitignore

I thought those are needed? why add them to gitignore now?

**Comment by AmiraVomir** [comment]
@carlitoswillis implemented those new lines, please re-review

**Comment by carlitoswillis** [comment]
lgtm!

**Pull request merged**